src/atomate2/forcefields/jobs.py

```diff
@@ -131,21 +188,19 @@ def make(
                 "Behavior may vary..."
             )
 
-        result = self._evaluate_static(structure)
+        static_calc = Relaxer(self._calculator(), relax_cell=False)
+        result = static_calc.relax(structure, steps=1)
```

Comment:
I think we should actually use the values of `self.relax_cell` and `self.steps` here. If the user really wants to turn a static into a relaxation, they should be able to.
```suggestion
        static_calc = Relaxer(self._calculator(), relax_cell=self.relax_cell)
        result = static_calc.relax(structure, steps=self.steps)
```

Reply:
Sounds good to me, I went back and forth on this. Also added a bit to the docstr to explain that defaults are set but they are not fixed